### PR TITLE
add: grid, list, modal, toast docs

### PR DESCRIPTION
**Pull request by nayyyhaa**
Added documentation for grid, list, modal, toast components.

### CODE REVIEW

File: docs/grid/index.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+          </div>
+          <h3 class="title">Code Snippets</h3>
+          <div class="container code-container colored-container m-v-4 col-flex">
+            <pre class="code-snipet m-v-1 p-1">
+              <code class="html fs-s">
+              <!-- 4 Column grid Layout -->
+              <div class="grid-4-col">
+                <div class="grid-item-one">Grid Item One</div>
+                <div class="grid-item-two">Grid Item Two</div>
+                <div class="grid-item-three">Grid Item Three</div>
```
Comment (venkateshwarans) [inline docs/grid/index.html]: a suggestion -  a common class to select the grid items could be useful. I see you have used `grid-ctr` . but I think it serves a different purpose.

File: styles/main.css [hunk 2]
```diff
@@ -25,6 +25,14 @@
   line-height: 4rem;
 }
 
+.p-t-2 {
+  padding-top: 2rem;
+}
+
+.code-snipet {
```
Comment (venkateshwarans) [inline styles/main.css]: snippet typo

**Review by venkateshwarans — APPROVED** [review]
Just one comment. Apart from that everything looks good. Great work.

File: docs/grid/index.html (hunk 1 above)
Comment (nayyyhaa) [reply docs/grid/index.html]: Sure, will remove grid-ctr and give a common class.

File: styles/main.css (hunk 2 above)
Comment (nayyyhaa) [reply styles/main.css]: yes, will change it.

**Comment by nayyyhaa** [comment]
> Just one comment. Apart from that everything looks good. Great work.

Done with the said changes. Thanks for reviewing! :D

**Review by anik31 — APPROVED** [review]
Everything looks great. Liked the readme.

**Comment by nayyyhaa** [comment]
Thanks @anik31 & @venkateshwarans for reviewing. Merging the changes.

**Pull request merged**